### ISSUE TITLE
About Test Sets and Validation Sets

### ISSUE THREAD

**Issue by xdyfazio** [fourson/Learning-to-dehaze-with-polarization#7]
Hi author, I found that there is only validation set and training set generation inside the code, no test set. May I ask if your results are tested directly using the validation set？
